packages/client/lib/Chain.js

```diff
@@ -197,8 +197,17 @@ export default class Chain {
    * @param {!string} from - The address from which the message is signed.
    * @return {Promise<string>} Resolves with a signed transaction.
    */
-  async sendTransaction (to, value, data, from) {
-    return this.client.getMethod('sendTransaction')(to, value, data, from)
+  async sendTransaction (to, value, data, from, extraOutputs) {
+    return this.client.getMethod('sendTransaction')(to, value, data, from, extraOutputs)
+  }
+
+  /**
+   * Create, sign & broad a transaction with multiple outputs.
+   * @param {string[]} outputs - to, value, data, from.
+   * @return {Promise<string>} Resolves with a signed transaction.
+   */
+  async sendBatchTransaction (outputs) {
```

Comment:
`outputs` is very bitcoin specific term. `transactions` would be more generic.

Reply:
Good point. Changed `outputs` to `transactions`